src/displayUI.js

```diff
@@ -15,6 +15,8 @@ export function displayOneCategory(categoryElement) {
   categoryItemSvg.addEventListener('click', (e) => {
     e.preventDefault();
     changeColor(categoryItemSvg);
+    instanceOfMainClass.markAsImportant(categoryElement);
```

Comment:
mark and save to local storage should be done under markAsImportant, so call that updateStorageCategories under that method, so from here you need only single call. Also change for links mark.